cmd/criticize/main.go

```diff
@@ -229,10 +233,23 @@ func (l *linter) checkFile(f *ast.File) {
 
 			for _, warn := range c.Check(f) {
 				l.foundIssues = true
-				pos := l.ctx.FileSet().Position(warn.Node.Pos())
-				log.Printf("%s: %s: %v\n", pos, c.Rule, warn.Text)
+				loc := l.ctx.FileSet().Position(warn.Node.Pos()).String()
+				if l.shorterErrLocation {
+					loc = shortenLocation(loc)
+				}
+				log.Printf("%s: %s: %v\n", loc, c.Rule, warn.Text)
 			}
 		}(c)
 	}
 	wg.Wait()
 }
+
+func shortenLocation(loc string) string {
+	if strings.HasPrefix(loc, build.Default.GOPATH) {
+		return strings.Replace(loc, build.Default.GOPATH, "$GOPATH", 1)
```

Comment:
Can be a switch:)

Reply:
Agreed.
Especially in a context that we're going to add more neat checks for switches.